.travis.yml

```diff
@@ -68,6 +68,7 @@ matrix:
     - rm -R node
     - tar -czf exokit-macos.tar.gz --exclude=".*" --exclude="*.tar.gz" *
     - ./exokit-codesign-macos.sh
+    - npm run test:ci
```

Comment:
Hm, I don't understand why this needs to be different from `appveyor.yml`.

Reply:
Forgot to update appveyor after I was working on Travis. They match now.